include/deal.II/base/config.h.in

```diff
@@ -205,10 +205,11 @@
 /* cmake/configure/configure_2_trilinos.cmake */
 #cmakedefine DEAL_II_TRILINOS_CXX_SUPPORTS_SACADO_COMPLEX_RAD
 #cmakedefine DEAL_II_TRILINOS_WITH_EPETRAEXT
+#cmakedefine DEAL_II_TRILINOS_WITH_MUELU
```

Comment:
Why MueLu?

Reply:
I can move this elsewhere but I alphabetized the Trilinos projects because only this one is out of order.